### PR TITLE
Use monospace for HostFolder and VM (english)

### PR DESCRIPTION
**Pull request by lightlace**
Use monospace for HostFolder and VM in the english Minikube setup guide.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-18748--kubernetes-io-master-staging.netlify.com/docs/setup/learning-environment/minikube/#mounted-host-folders

File: content/en/docs/setup/learning-environment/minikube.md [hunk 1]
```diff
@@ -405,13 +405,13 @@ Some drivers will mount a host folder within the VM so that you can easily share
...
-| --- | --- | --- | --- |
-| VirtualBox | Linux | /home | /hosthome |
-| VirtualBox | macOS | /Users | /Users |
-| VirtualBox | Windows | C://Users | /c/Users |
-| VMware Fusion | macOS | /Users | /mnt/hgfs/Users |
-| Xhyve | macOS | /Users | /Users |
+| Driver        | OS      | HostFolder   | VM                 |
+| --------------|---------|--------------|--------------------|
+| VirtualBox    | Linux   | ``/home``    | ``/hosthome``      |
+| VirtualBox    | macOS   | ``/Users``   | ``/Users``         |
```
Comment (kbhawkey) [inline content/en/docs/setup/learning-environment/minikube.md]: @lightlace , thanks for this PR. 
- Are two back ticks required to format the directory names in the table? Or could you use one back tick?
- Could you add a table caption shortcode for this table. The table should have a caption.
This page explains how to use the table caption shortcode.
https://deploy-preview-18748--kubernetes-io-master-staging.netlify.com/docs/contribute/style/hugo-shortcodes/#table-captions

For example, you could add something like:

```
{{< table caption = "Drivers, operating systems, mounted host folders" >}}
... the table

{{< /table */>}}

```

**Comment by kbhawkey** [comment]
See issue #18710 .

File: content/en/docs/setup/learning-environment/minikube.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/setup/learning-environment/minikube.md]: I'd expect to see single backticks.

**Comment by inductor** [comment]
@lightlace ping

**Comment by sftim** [comment]
@lightlace, would you be willing to revise this based on feedback?

**Comment by jimangel** [comment]
@lightlace I am closing this PR since it has been open since Jan 17 with no updates. Feel free to address the feedback and re-open. Thanks!

/close

**Pull request closed without merging**